cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/ReconcilerUtils.java

```diff
@@ -531,7 +530,7 @@ public static Future<String> trustedCertificates(Reconciliation reconciliation,
                         if (certificates.list().isEmpty()) {
                             return Future.succeededFuture();
                         } else {
-                            return Future.succeededFuture(String.join("\n", certificates.list()));
+                            return Future.succeededFuture(certificates.list());
```

Comment:
Does the list have a fixed order? That is important to make sure the hash does not change randomly.